api/hearings/index.ts

```diff
@@ -98,7 +99,23 @@ export async function submitHearingRequest(req: EnhancedRequest, res: Response,
   const reqBody = req.body;
   const markupPath: string = `${hearingsUrl}/hearing`;
   try {
-    const {status, data}: { status: number, data: ServiceHearingValuesModel } = await handlePost(markupPath, reqBody, req);
+    const { status, data }: { status: number, data: ServiceHearingValuesModel } = await handlePost(markupPath, reqBody, req);
+    res.status(status).send(data);
+  } catch (error) {
+    next(error);
+  }
+}
+
+/**
+ * cancelHearingRequest - cancel hearing request
+ */
+export async function cancelHearingRequest(req: EnhancedRequest, res: Response, next: NextFunction) {
+  const reqBody = req.body;
+  const caseId = req.query.caseId;
+  const markupPath: string = `${hearingsUrl}/cancelHearings/${caseId}`;
```

Comment:
This should be DELETE ${hearingsUrl}/hearing/${hearingId}, bear in mind it should be hearingId not caseId

Reply:
https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1515362177#ExpertUILowLevelDesignHearingsManagement-HMCInterface